apps/frontend/src/components/global/checklist/ChecklistRulesTable.vue

```diff
@@ -0,0 +1,246 @@
+<template>
+  <v-card height="94vh" overflow-auto>
+    <v-card-title class="pt-2">
+      <div
+        style="
+          display: flex;
+          align-items: center;
+          justify-content: space-between;
+          width: 100%;
+        "
+      >
+        <div>
+          <strong>
+            Rules ({{ numItems }} shown, {{ rules.length - numItems }} hidden)
+          </strong>
+        </div>
+        <div style="width: fit-content">
+          <v-switch
+            input-value="true"
+            style="margin-top: 0; padding-top: 0"
+            dense
+            inset
+            label="Short ID"
+            color="teal"
+            hide-details
+            @change="$emit('toggle-short-id')"
+          />
+        </div>
+      </div>
+      <v-spacer class="mt-0 pt-0" />
+      <v-select
+        v-model="selectedHeaders"
+        :items="headersList"
+        label="Select Columns"
+        class="mt-4 pt-0"
+        dark
+        item-color="white"
+        multiple
+        outlined
+        return-object
+        height="8.5vh"
+      >
+        <template #selection="{item, index}">
+          <div v-if="$vuetify.breakpoint.lgAndUp">
+            <v-chip v-if="index < 4" small>
+              <span>{{ item.text }}</span>
+            </v-chip>
+            <span v-if="index === 4" class="grey--text caption mt-0 pt-0">
+              (+{{ selectedHeaders.length - 4 }} others)
+            </span>
+          </div>
+          <div v-else>
+            <v-chip v-if="index < 3" small>
+              <span>{{ item.text }}</span>
+            </v-chip>
+            <span v-if="index === 3" class="grey--text caption ml-2">
+              (+{{ selectedHeaders.length - 3 }} others)
+            </span>
+          </div>
+        </template>
+      </v-select>
+    </v-card-title>
+    <v-card-text>
+      <!-- The mobile-breakpoint attribute fixes issues with the mobile resizing of a v-data-table-->
+      <v-data-table
+        :single-select="true"
+        disable-pagination
+        dense
+        mobile-breakpoint="0"
+        :items="rules"
+        :item-class="checkSelected"
+        :headers="headers"
+        hide-default-footer
+        class="overflow-auto"
+        height="68vh"
+        @current-items="getFiltered"
+        @click:row="showRule"
+      >
+        <template #[`item.status`]="{item}">
+          <v-chip :color="statusColor(item.status)" small
+            ><strong>{{ shortStatus(item.status) }}</strong>
+          </v-chip>
+        </template>
+        <template #[`item.ruleVersion`]="{item}">
+          {{ truncate(shortStigId(item.ruleVersion), 20) }}
```

Comment:
seems like we have a horizontal scroll bar on that table so i do not think we need to truncate the text esp when some columns are also doing line wrap

Reply:
Removed truncate